### PR TITLE
Features/user list

### PR DESCRIPTION
**Pull request by Las04ka**

### CODE REVIEW

File: src/app/shared/components/ag-grid/copy-cell/copy-cell.component.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+@Component({
+  selector: 'app-copy-cell',
+  templateUrl: './copy-cell.component.html',
+  styleUrls: ['./copy-cell.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class CopyCellComponent implements ICellRendererAngularComp {
+  valueToCopy!: string;
+
```
Comment (Anastasiia-Svechnikova) [inline src/app/shared/components/ag-grid/copy-cell/copy-cell.component.ts:13]: lets maybe call it email, it will look good in template

**Review by Anastasiia-Svechnikova — APPROVED** [review]

**Review by Anastasiia-Svechnikova — APPROVED** [review]

File: src/app/shared/components/confirm-modal/confirm-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { MatDialogRef } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-confirm-modal',
+  templateUrl: './confirm-modal.component.html',
+  styleUrls: ['./confirm-modal.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConfirmModalComponent {
+  @Output() confirmed = new EventEmitter<void>();
```
Comment (AleaJactaEst) [inline src/app/shared/components/confirm-modal/confirm-modal.component.ts:16]: hmm.. looks weird for me, is that from some best practice guide?
mostly different values passed to close, and then listen to afterClosed method

File: src/app/shared/components/confirm-modal/confirm-modal.component.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<h2
+  mat-dialog-title
+  class="mat-headline-5 title">
+  Confirm Action
```
Comment (AleaJactaEst) [inline src/app/shared/components/confirm-modal/confirm-modal.component.html:4]: I would like to make it more reusable and allow to populate with config like,
title, description, confrimBtnText, closeBtnText etc..

File: src/app/shared/directives/confirm-dialog.directive.ts [hunk 4]
```diff
@@ -1,23 +1,38 @@
...
   click(): void {
     this.confirmedSubscription = this.dialog
-      .open(ConfirmModalComponent)
-      .componentInstance.confirmed.subscribe(() => this.confirm.next());
+      .open(ConfirmModalComponent, {
+        data: { ...this.defaultConfig, ...this.modalConfig },
+      })
+      .afterClosed()
+      .subscribe((result) => {
+        if (result) this.confirm.next();
```
Comment (AleaJactaEst) [inline src/app/shared/directives/confirm-dialog.directive.ts:34]: let's use more functional style, RxJS filter instead if operator here

**Pull request merged**